### PR TITLE
WAI-930: update react scripts

### PR DESCRIPTION
**Pull request by edmofro**
### Issue #:
WAI-930

Same as https://github.com/beyondessential/tupaia/pull/3076, reopened for additional work on svg loading

### CODE REVIEW

**Review by edmofro — COMMENTED** [review]
Last few commits all look good to me! Only thing is whether we want to keep around the `OldIcons` folder. Do we need it? I'd prefer to just remember them in version control

File: .eslintrc [hunk 1]
```diff
@@ -118,6 +118,19 @@
       }
     },
     // Package overrides
+    {
+      "files": ["packages/admin-panel/**"],
+      "plugins": ["module-resolver"],
+      "rules": {
+        "react/forbid-prop-types": "off",
+        "import/no-cycle": "warn"
```
Comment (tcaiger) [inline .eslintrc]: This is removed in the following commit

**Comment by tcaiger** [comment]
@edmofro - this is looking nice and stable now. Would you mind having a quick review of the changes from Friday and today? And then I will move it to QA. Cheers

**Review by edmofro — COMMENTED** [review]
Yep I've checked 'em out and looks good. I can't actually click approve though!

**Comment by edmofro** [comment]
My comment re. Friday's changes [here](https://github.com/beyondessential/tupaia/pull/3169#pullrequestreview-752218015)

**Review by edmofro — COMMENTED** [review]
All good from my end :+1:

**Review by tcaiger — APPROVED** [review]

**Pull request merged**